apps/tools/blocks-crawler/src/main.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0
+
+use blocks::{check_block, fetch_block_info};
+use futures::{stream, StreamExt};
+use std::fs::create_dir_all;
+#[macro_use]
+extern crate log;
+
+mod blocks;
+
+#[tokio::main]
+async fn main() {
+    env_logger::init();
+    let concurrency = 42;
```

Comment:
Maybe make this an optional CLI argument with default value?  Not sure if 42 is the best for all network conditions

Reply:
fixed